test-project/helloFX/build.gradle

```diff
@@ -3,17 +3,18 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
     mavenCentral()
 }
 
 def target = System.getProperty("substrate.target")
 
+configurations {
+    provided
+}
+
 dependencies {
-    if ("ios" == target) {
-        implementation 'com.googlecode.plist:dd-plist:1.22'
-        implementation 'com.github.jnr:jnr-ffi:2.1.10'
-        implementation 'org.bouncycastle:bcpkix-jdk15on:1.49'
-    }
+    provided "com.gluonhq:substrate:0.0.2-SNAPSHOT"
```

Comment:
Are the other dependencies no longer needed on the classpath?

Reply:
These are transitive dependencies, and therefore, they are included in the classpath

Reply:
True.